README.md

```diff
@@ -109,60 +120,244 @@ Super similar to `compare_relations`, except it takes two select statements. Thi
 {% endset %}
 
 {{ audit_helper.compare_queries(
-    a_query=old_fct_orders_query,
-    b_query=new_fct_orders_query,
-    primary_key="order_id"
+    a_query = old_query,
+    b_query = new_query,
+    primary_key = "order_id"
 ) }}
 
+```
+
+## compare_row_counts ([source](macros/compare_row_counts.sql))
+This macro does a simple comparison of the row counts in two relations. 
+
+### Output:
+Calling this macro on two different relations will return a very simple table comparing the row counts in each relation. 
+
+| relation_name                                | total_records  |
+|----------------------------------------------|---------------:|
+| target_database.target_schema.my_a_relation  |     34,231     |
+| target_database.target_schema.my_b_relation  |     24,789     |
+
+### Usage:
+```sql
+
+{% set old_relation = adapter.get_relation(
+      database = "old_database",
+      schema = "old_schema",
+      identifier = "fct_orders"
+) -%}
+
+{% set dbt_relation = ref('fct_orders') %}
+
+{{ audit_helper.compare_row_counts(
+    a_relation = old_relation,
+    b_relation = dbt_relation
+) }}
 
 ```
 
 ### Arguments:
+* `a_relation` and `b_relation`: The [relations](https://docs.getdbt.com/reference/dbt-classes#relation) you want to compare. 
 
-* `a_query` and `b_query`: The queries you want to compare.
-* `primary_key` (optional): The primary key of the queries. Used to sort unmatched
-  results for row-by-row validation.
-* `summarize` (optional): Allows you to switch between a summary or detailed view
-  of the compared data. Accepts `true` or `false` values. Defaults to `true`.
-* `limit` (optional): Allows you to limit the number of rows returned when summarize=False. Defaults to `null` (no limit).
-
+# Compare Columns 
 ## compare_column_values ([source](macros/compare_column_values.sql))
```

Comment:
When would you ever want this query instead of compare_all_columns? Should we remove it altogether? It requires you to already know which column your problem is in, instead of helping you diagnose where the problem is. 

At minimum we probably want to swap them around and deemphasise it, but I sorta feel like we could remove it altogether.

Reply:
I disagree - I think this is a super valuable one! It does require you to know which column your problem is in (you could use the new `compare_which_columns_differ` for that, but it's way less computationally expensive to look at a single column vs. all of the columns!

Reply:
There is `compare_column_values_verbose` which we don't currently document, should we?